composables/utils/idb.ts

```diff
@@ -92,7 +93,7 @@ export class ObjectStore {
 }
 
 function request2promise(request: IDBRequest): Promise<any> {
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve) => {
```

Comment:
There's actually a bug here. 😅 The `onerror` callback should call `reject()` instead of `resolve()`.